src/utils/objects.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Checks if the argument is typeof object
+ *
+ * @param {*} obj
+ * @returns {Boolean}
+ */
+const isObject = (obj) =>
+  !!(obj && !Array.isArray(obj) && typeof obj === "object");
```

Comment:
This is really slick. I wouldn't have thought to include the check for `!Array.isArray(obj)`. I just pulled up a Node REPL to see how it worked.